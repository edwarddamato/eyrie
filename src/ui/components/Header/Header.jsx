import React from 'react';
import './Header.scss';
const app = require('../../../../.app.json');

const Header = () => (
  <header className="header">
    <h1 className="header_title">eyrie {app.version}</h1>
    <div className="header_author">Built with ❤️ by {app.author}</div>
  </header>
);

export default Header;
