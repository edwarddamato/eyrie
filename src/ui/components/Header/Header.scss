@import "../../styles/base";

.header {
  background-color: $white;
  padding: .5rem;
  overflow: auto;

  &_title {
    float: left;
    font-size: 1rem;
    font-weight: 100;
  }

  &_author {
    font-size: .7rem;
    float: right;
    color: $grey;
  }
}