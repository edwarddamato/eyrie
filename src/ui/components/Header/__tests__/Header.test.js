import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../';

describe('Header', () => {
  it('renders correctly', () => {
    const header = renderer.create(
      <Header />
    ).toJSON();
    expect(header).toMatchSnapshot();
  });
});
