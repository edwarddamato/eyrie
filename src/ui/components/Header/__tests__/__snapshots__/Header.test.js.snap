// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="header"
>
  <h1
    className="header_title"
  >
    eyrie
  </h1>
</header>
`;
